.github/workflows/container.yml

```diff
@@ -69,6 +74,12 @@ jobs:
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v3
+        with:
+          registry: ghcr.io
+          username: ${{ github.action }}
+          password: ${{ github.token }}
```

Comment:
```suggestion
      - name: Login to GitHub Container Registry
        if: github.event_name == 'pull_request'
        uses: docker/login-action@v3
        with:
          registry: ghcr.io
          username: ${{ github.action }}
          password: ${{ github.token }}
```

Reply:
Or do you want to upload the image to both registries?

Reply:
Yes that's my intention :-)